examples/python/rrt-star/main.py

```diff
@@ -0,0 +1,280 @@
+            rr.AnyValues(cost=[float(node.cost) for node in tree]),
+        )
+        rr.log("map/new/random_point", rr.Points2D([random_point], radii=0.008))
+        rr.log("map/new/closest_node", rr.Points2D([closest_node.pos], radii=0.008))
+        rr.log("map/new/new_point", rr.Points2D([new_point], radii=0.008))
+
+        color = np.array([0, 255, 0, 255]).astype(np.uint8)
+        if intersects_obs:
+            color = np.array([255, 0, 0, 255]).astype(np.uint8)
+        rr.log(
+            "map/new/new_edge",
+            rr.LineStrips2D([(closest_node.pos, new_point)], colors=[color], radii=0.001),
+        )
+
+        if not intersects_obs:
+            # Searches for the point in a neighborhood that would result in the minimal cost (distance from start).
+            close_nodes = tree.in_neighborhood(new_point, neighborhood_size)
+            rr.log("map/new/close_nodes", rr.Points2D([node.pos for node in close_nodes]))
+
+            min_node = min(
+                filter(
+                    lambda node: not mp.intersects_obstacle(node.pos, new_point),
+                    close_nodes + [closest_node],
+                ),
+                key=lambda node: node.cost + distance(node.pos, new_point),
+            )
+
+            cost = distance(min_node.pos, new_point)
+            added_node = Node(min_node, new_point, cost + min_node.cost)
+            tree.add_node(min_node, added_node)
+
+            # Modifies nearby nodes that would be reached faster by going through `added_node`.
+            for node in close_nodes:
+                cost = added_node.cost + distance(added_node.pos, node.pos)
+                if not mp.intersects_obstacle(new_point, node.pos) and cost < node.cost:
+                    parent = node.parent
+                    if parent is not None:
+                        parent.children.remove(node)
+
+                        node.parent = added_node
+                        node.change_cost(cost - node.cost)
+                        added_node.children.append(node)
+
+            if (
+                distance(new_point, end) < max_step_size
+                and not mp.intersects_obstacle(new_point, end)
+                and end_node is None
+            ):
+                end_node = Node(added_node, end, added_node.cost + distance(new_point, end))
+                tree.add_node(added_node, end_node)
+                step_found = step
+
+            if end_node:
+                # Reconstruct shortest path in tree
+                path = path_to_root(end_node)
+                segments = [(path[i], path[i + 1]) for i in range(len(path) - 1)]
+                rr.log(
+                    "map/path",
+                    rr.LineStrips2D(segments, radii=0.002, colors=[0, 255, 255, 255]),
+                )
+
+    return path
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(description="Example of using the Rerun visualizer")
```

Comment:
we can do a bit better than that :). Please describe here what the application does

Reply:
Oops, I forgot to change that after copying the template. Fixed now.